README.md

```
@@ -85,10 +85,10 @@ Create an `Authorizer` object and authorize the request with the policy set and
 authorizer = CedarPolicy::Authorizer.new
 ```
 
-If boolean result is enough, use `#authorize?` method.
+If boolean result is enough, use `#authorized?` method.
 
 ```ruby
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)